docs: add ngVikings 2019 to events page

softchris opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
ngVikings2019 entry missing in events.html

## What is the new behavior?
ngVikings2019 entry added to events.html

## Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

softchris committed:
docs: add ngVikings 2019 to events page

StephenFluin reviewed (changes requested):
Can you make sure the commit message matches our contributing guidelines?

I'm also seeing a lot of unrelated whitespace changes. Can you make sure you only touch the new event, and that it matches the existing formatting?

softchris commented:
> Can you make sure the commit message matches our contributing guidelines?
> 
> I'm also seeing a lot of unrelated whitespace changes. Can you make sure you only touch the new event, and that it matches the existing formatting?

changed the title, hope that's the format you mean. Also fixed the formatting I think..

trotyl commented on aio/content/marketing/events.html:
```diff
@@ -25,6 +25,12 @@ <h1 class="banner-headline no-toc no-anchor">Events</h1>
+    <tr>
+      <th><a href="https://ngvikings.org/" title="ngVikings">ngVikings</a></th>
+      <td>Copenhagen, Denmark</td>
+      <td>May 26-28, 2019</td>
```
In the website it's said to be **May 27-28th, 2019**, better to align them?

softchris replied:
well 26th is workshop day, 27-28 is conference.. Yes we will update the web page to say that :)

StephenFluin commented:
Now I think you'll need to squash the commits. 

https://twitter.com/stephenfluin/status/1009904095073718275

softchris commented:
> Now I think you'll need to squash the commits.
> 
> https://twitter.com/stephenfluin/status/1009904095073718275

ok I squashed it.. Happy now? :)

brandonroberts commented:
The git commit message needs to be updated to `docs: add ngVikings 2019 to events page`

webmaxru commented:
@softchris any status on changing commit message?

mary-poppins commented:
You can preview 712a89c at https://pr28501-712a89c.ngbuilds.io/.
You can preview 239ffe2 at https://pr28501-239ffe2.ngbuilds.io/.
You can preview 100e5c2 at https://pr28501-100e5c2.ngbuilds.io/.
You can preview b931a7f at https://pr28501-b931a7f.ngbuilds.io/.
You can preview a8c5459 at https://pr28501-a8c5459.ngbuilds.io/.
You can preview f186afd at https://pr28501-f186afd.ngbuilds.io/.
You can preview 9b06a58 at https://pr28501-9b06a58.ngbuilds.io/.

mary-poppins commented:
You can preview 86b1be5 at https://pr28501-86b1be5.ngbuilds.io/.

mary-poppins commented:
You can preview 3aba186 at https://pr28501-3aba186.ngbuilds.io/.

brandonroberts reviewed (approved):
LGTM. @StephenFluin will give final approval.
